app/src/main/AndroidManifest.xml

```diff
@@ -2,23 +2,33 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.literacyapp.battery">
 
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
+
     <application
+        android:fullBackupContent="true"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity
-            android:name=".BatteryActivity"
+
+        <activity android:name=".BatteryActivity"
             android:label="@string/app_name">
             <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
-                <!-- Check if the device is connected to usb or not-->
+            </intent-filter>
+        </activity>
+
+        <receiver
+            android:name=".BatteryReciever">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
```

Comment:
Would it be better to listen for `android.intent.action.BATTERY_LOW` intents? - https://developer.android.com/training/monitoring-device-state/battery-monitoring.html